libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md

```diff
@@ -107,6 +137,27 @@ __Example 2__ shows how you can create a __PdfExportSettings__ object and assign
 
 {{endregion}}
 
+__Example 3__ shows how you can create a __PdfExportSettings__ object with a different set of settings. The settings used specify an OwnerPassword and UserAccessPermissions.
+
+
+#### __[C#] Example 3: Export settings__
+
+{{region cs-radpdfprocessing-formats-and-conversion-pdf-settings_1}}
```

Comment:
Leaving a blank line after the region opening tag makes the code snippet easier to read. Also make sure the code is formatted correctly.